dashboard/test/helpers/plc_evaluation_helper_test.rb

```diff
@@ -0,0 +1,120 @@
+
+    q1_dsl = <<DSL
+    name 'Question 1'
+    question 'What is your name?'
+    answer 'Sir Lancelot', weight: 1, stage_name: '#{@stage_honesty.name}'
+    answer 'Sir Robin', weight: 1, stage_name: '#{@stage_no_nickname.name}'
+    answer 'Sir Galahad', weight: 1, stage_name: '#{@stage_honesty.name}'
+    answer 'King Arthur', weight: 1, stage_name: '#{@stage_honesty.name}'
+    answer 'Mr Edgecase'
+DSL
+
+    q2_dsl = <<DSL
+    name 'Question 2'
+    question 'What is your quest?'
+    answer 'I seek the grail', weight: 1, stage_name: '#{@stage_honesty.name}'
+    answer 'Yes, yes, I seek the Grail', weight: 1, stage_name: '#{@stage_no_nickname.name}'
+    answer 'I seek something else'
+DSL
+
+    q3_dsl = <<DSL
+    name 'Question 3'
+    question 'What is your favorite color?'
+    answer 'Blue', weight: 1, stage_name: '#{@stage_blue.name}'
+    answer 'Yellow - no, blue', weight: 1, stage_name: '#{@stage_cliffs.name}'
+    answer 'No preference'
+DSL
+
+    q4_dsl = <<DSL
+    name 'Question 4'
+    question 'What is the capital of Assyria?'
+    answer 'I dont know that!', weight: 1, stage_name: '#{@stage_ignorance.name}'
+    answer 'Nineveh'
+DSL
+
+    q5_dsl = <<DSL
+    name 'Question 5'
+    question 'What is the airspeed velocity of an unladen swallow?'
+    answer 'What do you mean, an African or European Swallow?', weight: 1, stage_name: '#{@stage_ornithology.name}'
+    answer '15 m/s'
+DSL
+
+    @evaluation_multi1 = EvaluationMulti.create_from_level_builder({name: 'Question 1'}, {dsl_text: q1_dsl})
+    @evaluation_multi2 = EvaluationMulti.create_from_level_builder({name: 'Question 2'}, {dsl_text: q2_dsl})
+    @evaluation_multi3 = EvaluationMulti.create_from_level_builder({name: 'Question 3'}, {dsl_text: q3_dsl})
+    @evaluation_multi4 = EvaluationMulti.create_from_level_builder({name: 'Question 4'}, {dsl_text: q4_dsl})
+    @evaluation_multi5 = EvaluationMulti.create_from_level_builder({name: 'Question 5'}, {dsl_text: q5_dsl})
+
+    levelgroup_dsl = <<DSL
+    name 'Crossing the bridge of death'
+    title 'Bridge of Death Evaluation'
+    submittable 'true'
+
+    page
+    level 'Question 1'
+    level 'Question 2'
+    level 'Question 3'
+    level 'Question 4'
+    level 'Question 5'
+DSL
+
+    @evaluation = LevelGroup.create_from_level_builder({name: 'evaluation'}, {dsl_text: levelgroup_dsl})
+  end
+
+  test 'submit evaluation enrolls user in appropriate modules' do
+    assert_equal [@module_blue.id, @module_honesty.id], get_preferred_modules_for_answers([[@evaluation_multi1, 'Sir Lancelot'],
+                                                                                           [@evaluation_multi2, 'I seek the grail'],
+                                                                                           [@evaluation_multi3, 'Blue']])
```

Comment:
Indented?  Expecting something like:
```
assert_equal [@module_blue.id, @module_honesty.id], get_preferred_modules_for_answers([
  [@evaluation_multi1, 'Sir Lancelot'],
  [@evaluation_multi2, 'I seek the grail'],
  [@evaluation_multi3, 'Blue']
])
```
(This isn't part of the style guide so feel free to ignore.)